src/lib/stitching/exchange/v2/stitching.ts

```diff
@@ -378,13 +392,35 @@ export const exchangeStitchingEnvironment = ({
         // The money helper resolvers
         ...totalsResolvers("CommerceBuyOrder", orderTotals),
         buyerDetails: buyerDetailsResolver,
+        buyerProfile: {
+          fragment: gql`... on CommerceBuyOrder {
```

Comment:
This _probably_ should be `on CommerceOrder`, but should test that on an order of that type.